deploy/ml-config.xml

```diff
@@ -242,15 +242,15 @@
         <!--
         Sample for:
           <name xmlns="http://marklogic.com/ns/sample">bob<name>
-
+-->
         <range-element-index>
           <scalar-type>string</scalar-type>
-          <namespace-uri>http://marklogic.com/ns/sample</namespace-uri>
-          <localname>name</localname>
+          <namespace-uri></namespace-uri>
+          <localname>region</localname>
```

Comment:
It was needed for something; I think it was for a jsearch tutorial. I will remove.